src/hotspot/share/gc/parallel/psScavenge.cpp

```diff
@@ -553,7 +553,7 @@ bool PSScavenge::invoke_no_policy() {
                                                            _tenuring_threshold,
                                                            survivor_limit);
 
-       log_debug(gc, age)("Desired survivor size " SIZE_FORMAT " bytes, new threshold %u (max threshold " UINTX_FORMAT ")",
+       log_debug(gc, age)("Desired survivor size " SIZE_FORMAT " bytes, new threshold %u (max threshold %u)",
```

Comment:
Consider also, instead of SIZE_FORMAT we can use "%zu".

Reply:
`%zu` is used instead.